src/Components/WorkloadRules/WorkloadRules.js

```diff
@@ -198,6 +227,17 @@ const WorkloadRules = ({ workload }) => {
     },
   };
 
+  const onSort = (_e, index, direction) => {
+    setRowsFiltered(false);
```

Comment:
Can you please clarify what we do with `rowsFiltered` in the component after we set it to false here? I can't find any place where the flag is set back to true (`setRowsFiltered(true)`). Does it mean that `loadingState` variable will be true?

Reply:
Regarding that, we setting it to true on the line 76